sso-plugin/optimize-sso-keycloak/elasticsearch/elasticsearch.yml

```diff
@@ -0,0 +1,93 @@
+# ======================== Elasticsearch Configuration =========================
```

Comment:
can't we just run it with default config?

Reply:
we could do that, i just added the watermark config directly, maybe they are already environment variables?